packages/calculator/src/draggable-calculator.jsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import Calculator from '@pie-framework/material-ui-calculator';
+import Draggable from 'react-draggable';
+import Typography from 'material-ui/Typography';
+import Close from 'material-ui-icons/Close';
+import IconButton from 'material-ui/IconButton';
+import PropTypes from 'prop-types';
+
+const classes = {
+  handle: {
+    border: '1px solid black',
+    zIndex: '10',
+    position: 'absolute',
+    cursor: 'move'
+  },
+  typography: {
+    fontSize: '20px',
+    width: '100%',
+    textAlign: 'center',
+    borderBottom: '1px solid black',
+    fontWeight: 'bold',
+    backgroundColor: '#f3f3f3',
+    display: 'block'
+  },
+  closeIcon: {
+    verticalAlign: 'middle',
+    cursor: 'pointer'
+  }
+};
+
+
+class DraggableCalculator extends React.Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      deltaPosition: {
+        x: 0, y: 0
+      }
+    }
+  }
+
+  handleDrag = (e, ui) => {
+    const { x, y } = this.state.deltaPosition;
+    this.setState({
+      deltaPosition: {
+        x: x + ui.deltaX,
+        y: y + ui.deltaY,
+      }
+    });
+  }
+
+  render() {
+    const { mode, show, closeHandler } = this.props;
+    const { x, y } = this.state.deltaPosition;
+    return show ?
+      (<Draggable
+        onDrag={this.handleDrag}
+        defaultPosition={{ x, y }}>
+        <div style={classes.handle}>
```

Comment:
`className={classes.handle}` not `style={classes.handle}`
Also you probably will want to re-jig the node structure to accommodate the handle node.

```jsx
<div className={'handle' + ... }>
  <Typography>Calculator</Typography>
  <CloseButton/>
</div>
```
Also can you make the button right aligned and the title center aligned in the title bar?